src/DynamoCoreWpf/Views/LibrarySearchView.xaml.cs

```diff
@@ -379,7 +394,7 @@ private void OnCategoryContentKeyDown(object sender, KeyEventArgs e)
 
                 // Otherwise user pressed down, we have to move to first member button.
                 var memberGroupsListBox = WpfUtilities.ChildOfType<ListBox>(searchCategoryElement, "MemberGroupsListBox");
-                var listItem = FindFirstChildListItem(memberGroupsListBox, "MembersListBox");
+                var listItem = FindChildListItemByIndex(memberGroupsListBox, "MembersListBox");
```

Comment:
Sorry I was reviewing half way before I had to attend to something else, thanks for the clarification.